tests/test_databases.py

```diff
@@ -23,7 +23,7 @@ def mysql_versions(wrapped_func):
     """
 
     @functools.wraps(wrapped_func)
-    def check(*args, **kwargs):
+    def check(*args, **kwargs):  # pragma: no cover
```

Comment:
Ah because of
```
[coverage:run]
source = databases, tests
```
Why run coverage on tests? I wonder

Reply:
Not sure, it was like that.
I guess it ensures all tests are run? If you run it for all backends at once each test has to have to run at least once. So it might discover some missing config (URL etc.) that you should add if some of the tests are always skipped (or refactor)

Reply:
I think this is the same pattern in all Encode projects. In my experience it makes sure the tests are covered by the tests too (?).
So for example if you have unreachable code or branching in your tests, this will catch it.

Reply:
ok, leaving as it is